src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1269,7 +1266,7 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
     val workspaceName = workspaceRequest.toWorkspaceName
     requireCreateWorkspaceAccess(workspaceName, txn) {
       containerDAO.workspaceDAO.loadContext(workspaceName, txn) match {
-        case Some(_) => Future.successful(PerRequest.RequestComplete(ErrorReport(StatusCodes.Conflict, s"Workspace ${workspaceRequest.namespace}/${workspaceRequest.name} already exists")))
+        case Some(_) => Future.successful(RequestComplete(ErrorReport(StatusCodes.Conflict, s"Workspace ${workspaceRequest.namespace}/${workspaceRequest.name} already exists")))
```

Comment:
Originally I decided that since it wasn't directly inside of a transaction, it didn't need to be an exception, but I might be wrong